src/test/kotlin/com/terraformation/backend/accelerator/db/VoteStoreTest.kt

```diff
@@ -0,0 +1,592 @@
+package com.terraformation.backend.accelerator.db
+
+import com.terraformation.backend.RunsAsUser
+import com.terraformation.backend.TestClock
+import com.terraformation.backend.accelerator.model.VoteModel
+import com.terraformation.backend.db.DatabaseTest
+import com.terraformation.backend.db.accelerator.CohortPhase
+import com.terraformation.backend.db.accelerator.VoteOption
+import com.terraformation.backend.db.accelerator.tables.pojos.ProjectVotesRow
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.UserId
+import com.terraformation.backend.mockUser
+import java.time.Instant
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+
+class VoteStoreTest : DatabaseTest(), RunsAsUser {
+  override val user = mockUser()
+
+  private val clock = TestClock()
+  private val store: VoteStore by lazy { VoteStore(clock, dslContext, projectVotesDao) }
+
+  data class VoteKey(val userId: UserId, val projectId: ProjectId, val phase: CohortPhase)
+
+  @BeforeEach
+  fun setUp() {
+    insertUser()
+    insertOrganization()
+  }
+
+  @Nested
+  inner class Create {
+    @Test
+    fun `creates blank vote for self`() {
+      val projectId = insertProject()
+
+      clock.instant = Instant.EPOCH.plusSeconds(500)
+      val phase: CohortPhase = CohortPhase.Phase0DueDiligence
+      store.create(VoteModel.create(projectId, phase, user.userId, null, null))
+
+      assertEquals(
+          listOf(
+              ProjectVotesRow(
+                  projectId = projectId,
+                  phaseId = phase,
+                  userId = user.userId,
+                  voteOptionId = null,
+                  conditionalInfo = null,
```

Comment:
Totally okay to include these for explicitness in the test, but FYI, in case you didn't notice, the `Row` classes all have null defaults for all their properties.